features/promotion/receiving_discount_based_on_number_of_items_from_specific_taxon.feature

```diff
@@ -0,0 +1,55 @@
+@promotion
+Feature: Receiving discount based on number of products from specific taxon
```

Comment:
Receiving **a** discount based on number of products from **the** specific taxon